src/test/java/io/strimzi/kafka/bridge/http/ConsumerGeneratedNameTest.java

```diff
@@ -37,9 +43,11 @@ static void revertUnsetBridgeID() {
     }
 
     @Test
-    void createConsumerNameNotSet(VertxTestContext context) {
+    void createConsumerNameNotSet(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
         JsonObject json = new JsonObject();
+        AtomicReference<String> name = new AtomicReference<>();
```

Comment:
can't we just use a field member as we do for `ConsumerTest` class?

```java
private String name:
```

Reply:
I am not sure what do you mean :( the name is generated, thus it is something like `kafka-bridge-consumer-564ecf654ef54c65a564ef`. We do not know the generated name before the test is executed.